### PR TITLE
wordgrinder 0.7.2 (new formula)

### PR DESCRIPTION
**Pull request by ak2k**
- [yes] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [yes] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [yes] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [yes] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [yes] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
The included test is not a great one; however, there is a test included.

### CODE REVIEW

**Comment by ak2k** [comment]
If anyone has any guidance to offer on the likely cause of this bottling error, I would appreciate it.

**Comment by SMillerDev** [comment]
The install is empty on high sierra: https://jenkins.brew.sh/job/Homebrew%20Core%20Pull%20Requests/44953/

might be a fluke.

**Comment by SMillerDev** [comment]
@BrewTestBot test this please

**Comment by fxcoudert** [comment]
Make install is putting files in the wrong directory:

```
==> make install PREFIX=/usr/local/Cellar/wordgrinder/0.7.2
[1/3] install -m 644 bin/wordgrinder.1 /private/tmp/wordgrinder-20190717-26740-d1hm4u/wordgrinder-0.7.2/.brew_home/share/man/man1/wordgrinder.1
[2/3] install -m 644 README.wg /private/tmp/wordgrinder-20190717-26740-d1hm4u/wordgrinder-0.7.2/.brew_home/share/doc/wordgrinder/README.wg
[3/3] install -m 755 bin/wordgrinder-builtin-curses-release-stripped /private/tmp/wordgrinder-20190717-26740-d1hm4u/wordgrinder-0.7.2/.brew_home/bin/wordgrinder
 ```

**Comment by ak2k** [comment]
@BrewTestBot test this please

**Comment by SMillerDev** [comment]
Still the wrong directory.

**Comment by SMillerDev** [comment]
@brewtestbot test this please

**Comment by ak2k** [comment]
@BrewTestBot test this please

**Comment by ak2k** [comment]
@BrewTestBot test this please

**Comment by ak2k** [comment]
Are there any changes required prior to inclusion of this new formula?

**Comment by fxcoudert** [comment]
Looks good to me. Can you please squash everything into a single commit, with message `wordgrinder 0.7.2 (new formula)`? We should then be ready to merge.

File: Formula/wordgrinder.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+class Wordgrinder < Formula
+  desc "Unicode-aware word processor that runs in a terminal"
+  homepage "https://cowlark.com/wordgrinder"
+  url "https://github.com/davidgiven/wordgrinder/archive/0.7.2.tar.gz"
+  sha256 "4e1bc659403f98479fe8619655f901c8c03eb87743374548b4d20a41d31d1dff"
+  head "https://github.com/davidgiven/wordgrinder.git"
+
+  depends_on "lua"
+  depends_on "ninja"
```
Comment (fxcoudert) [inline Formula/wordgrinder.rb]:
```suggestion
  depends_on "ninja"  => :build
```

This is a build-only dependency, and should be moved above `lua`.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

**Comment by chenrui333** [comment]
@ak2k Any update on this PR? :)

**Comment by iMichka** [comment]
This has been open since July 11, and has not moved since 26 days. I'll close this to reduce the number of open issues in our bug tracker. Please submit the formula again, and link to this PR.

**Pull request closed without merging**